documentation/modules/operators/con-topic-operator-store.adoc

```diff
@@ -0,0 +1,115 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-topic-operator.adoc
+
+[id='ref-topic-operator-store-{context}']
+= Topic Operator topic store
+
+[role="_abstract"]
+The Topic Operator uses persistent storage to store topic metadata describing topic configuration as key-value pairs.
+The _topic store_ is based on the Kafka Streams key-value mechanism, which uses persistent storage with local and remote components.
+
+Topic metadata is stored and accessed locally in-memory.
+Updates from operations applied to the local in-memory topic store are persisted to a backup topic store on disk.
```

Comment:
This is again confusing since the previous paragraph just explained how it uses persistent storage and here it suddenly said it is in-memory. Maybe this would be better?

```suggestion
Topic metadata are cached in-memory and accessed locally.
Updates from operations applied to the local in-memory cache are persisted to a backup topic store on disk.
```

Reply:
Thanks. _Cache_ helps here.